src/callbacks_step/analysis_dg1d.jl

```diff
@@ -66,16 +67,20 @@ function calc_error_norms(func, u, t, analyzer,
     for i in eachnode(analyzer)
       u_exact = initial_condition(get_node_coords(x_local, equations, dg, i), t, equations)
       diff = func(u_exact, equations) - func(get_node_vars(u_local, equations, dg, i), equations)
-      l2_error += diff.^2 * (weights[i] * jacobian_local[i])
+
+      l2_error  += diff.^2 * (weights[i] * jacobian_local[i])
       linf_error = @. max(linf_error, abs(diff))
+      l1_error  += abs.(diff) * (weights[i] * jacobian_local[i])
```

Comment:
Not sure if `@. abs(diff) * (weights[i] * weights[j] * jacobian_local[i, j])` is advantageous here/whether it makes a difference at all.